179: Submission button and checker function

ajtadeo opened the pull request:
## Summary

Closes #179 

- created check button for each Question component
- display check button and incorrect text if answer is incorrect
- display drop down button (expanded by default), correct text, and set icon color if answer is correct
- update global.scss colors

## Screenshots

<img width="1223" alt="image" src="https://user-images.githubusercontent.com/76643809/222668712-bd51f572-ab0d-49c4-85b2-d9b212e44100.png">

<img width="1174" alt="image" src="https://user-images.githubusercontent.com/76643809/222668783-091a4ba2-d59b-4d3f-aa93-aa79296fb7f4.png">

ajtadeo committed:
check button and expand funcionality

ajtadeo committed:
lint fix

ajtadeo committed:
fix lint

ajtadeo committed:
lint fix

ajtadeo committed:
fix question shadow

ajtadeo committed:
uncomment setCorrect() to view correct text

ajtadeo committed:
cursor: pointer on dropdown button

ajtadeo committed:
lint fix

ajtadeo committed:
remove complete attribute from Question, unneeded

sweeneyngo committed:
Added styling todos

sweeneyngo committed:
Fixed lint

sweeneyngo reviewed (dismissed):
→ styling fix:

`Question.tsx`
- check button: check is lowercase, fix center padding

`Exercise1.tsx`
- Missing lightbulb SVG (green/red)
- FIX, font-size: 30px, line-height: 39px
- B lowercase, no period

(refs from Figma)
![image](https://user-images.githubusercontent.com/44306479/222853069-930b648b-b015-4921-aa8b-1feac8973984.png)
![image](https://user-images.githubusercontent.com/44306479/222853092-4d8606ea-74eb-4a75-8488-ae2be556530d.png)
![image](https://user-images.githubusercontent.com/44306479/222853097-535035f8-f334-404f-bd4b-bf51b6735f8c.png)

sweeneyngo committed:
Fixed lint

Dankco committed:
Merge branch 'main' into 179-check-button

Dankco committed:
Merge branch 'main' into 179-check-button

Dankco commented:
This PR is done right?

ajtadeo committed:
fix styling, add svg icon

ajtadeo commented:
- fixed capitalization of check button, fixed center padding
- added SVG for feedback text
- changed question from h3 -> h2 for correct font size
- made box names lowercase

Dankco committed:
Merge branch 'main' into 179-check-button
